<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="expTab">
         <attribute name="title">
          <string>Experiment</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Experimentanzahl :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="numExpRunsSpinBox">
              <property name="minimum">
               <number>10</number>
              </property>
              <property name="maximum">
               <number>1000</number>
              </property>
              <property name="value">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <spacer name="horizontalSpacer_13">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Proband*in :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="liedProband">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>0</height>
               </size>
              </property>
              <property name="maxLength">
               <number>32</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pubuProband">
              <property name="text">
               <string>festlegen</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_14">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>103</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="stroopStartButton">
              <property name="minimumSize">
               <size>
                <width>400</width>
                <height>200</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>28</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Start Stroop</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>103</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="resultsTab">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <attribute name="title">
          <string>Ergebnisse</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout" rowstretch="1" columnstretch="1">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <item row="0" column="0">
           <widget class="QTableWidget" name="resultsTableWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>1</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
            <property name="rowCount">
             <number>0</number>
            </property>
            <property name="columnCount">
             <number>6</number>
            </property>
            <attribute name="verticalHeaderCascadingSectionResizes">
             <bool>true</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Modus</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Text</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Angezeigte Farbe</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Gewählte Farbe</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Übereinstimmung</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Reaktionszeit</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="actionExportCSVStats"/>
    <addaction name="separator"/>
    <addaction name="actionExportCSV"/>
    <addaction name="actionExportAllCSV"/>
   </widget>
   <widget class="QMenu" name="menuModus">
    <property name="title">
     <string>Modus</string>
    </property>
    <addaction name="actionEvalAllTrials"/>
    <addaction name="actionEvalCorrectTrials"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuModus"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave">
   <property name="text">
    <string>Experimentdatei speichern unter...</string>
   </property>
  </action>
  <action name="actionExportCSV">
   <property name="text">
    <string>Exportiere CSV...</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Experimentdatei laden...</string>
   </property>
  </action>
  <action name="actionEvalAllTrials">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Alle Antworten evaluieren</string>
   </property>
  </action>
  <action name="actionEvalCorrectTrials">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Nur korrekte Antworten evaluieren</string>
   </property>
  </action>
  <action name="actionExportCSVStats">
   <property name="text">
    <string>Exportiere CSV mit Statistik...</string>
   </property>
  </action>
  <action name="actionExportAllCSV">
   <property name="text">
    <string>Exportiere alle Experimente nach CSV...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
